### ISSUE TITLE
Is this in the iOS App Store yet?

### ISSUE THREAD

**Issue by mgoose99** [Schlaubischlump/Cookbook-Client#2]
The screenshots look great. Is this in the app store yet? I can't seem to find it.

**Comment by Schlaubischlump** [comment]
No, and most likely it won't be for the foreseeable future. There are still some rough edges I want to fix, but unfortunately I don't have the time to fix the app and keep it up to date (which is an official AppStore policy if you want to keep your app in the app store).

This could change in the future if I have more time or if someone is willing to help with the development.

**Comment by Teifun2** [comment]
@Schlaubischlump Hey fellow app developer.
I'm also working on an nextcloud cookbook app. https://github.com/Teifun2/nextcloud-cookbook-flutter
The App is developed with flutter and should therefore be available for IOS and Android there should even be support for semi native desktop apps for linux / windows / and macOS.

If we could somehow team up then maintaining and improving the app could be much faster as it will have an effect for all platforms. 

Additionally i was wondering if you have an IOS developers license!

If you are in any way interested please hit me up!

**Comment by Schlaubischlump** [comment]
Hi @Teifun2

I'm currently writing my master thesis and therefore don't really have time to work on as many side projects as I would like to. Besides that I'm not familiar with Flutter, so it would take additional time to learn it. 
That said, I would really like to help with the project, but it might take a couple of month until I have enough free time to get started learning Flutter and reading your code. 

Regarding the Developer License: I don't have one yet, but I'm planing on buying a membership. 

PS: I saw you forked a project for "Die Staemme". I assume the game is only popular in german speaking countries. So if you speak german by any chance, it might be easier to communicate in german going on.

**Comment by Teifun2** [comment]
@Schlaubischlump Korrekt ich spreche deutsch. Magst du mich über den nextcloud-cookbook matrix (element)  chat kontaktieren? Dann müssen wir nicht dieses Issue hier voll spamen. Du kannst mich dort unter dem gleichen namen finden und direkt anschreiben.

**Comment by wieluk** [comment]
Also kann ich die app nicht ohne gejailbraktes Ios gerät installieren oder?

**Comment by Schlaubischlump** [comment]
Doch. Wenn du sie selber über Xcode installierst. Wenn du eine normale Apple ID hast musst du das alle 7 Tage neu machen. Mit einem Developer ACC hält sie so lange wie du Entwickler bist. Habe selber keinen Developer account, deswegen steht sie nicht im App Store

**Comment by aniqueta** [comment]
@Schlaubischlump This looks like almost a v1 already. :) I'd enjoy testing when you're ready, and I'm happy to help by contributing to your Apple Developer Account if needed. As a macOS and iOS user, my preference would be a native app instead of a Flutter app. But something is better than nothing, and Flutter might be the easiest and most efficient path forward for a side project.

**Comment by Schlaubischlump** [comment]
Currently I don't plan to publish this app in the App store for several reasons: 
1. I don't have the time to keep this application updated. Maintaining the LocationSimulator application is more than enough work for my freetime. Apple requires you to keep the app updated. I can not meet this requirement.  
2. The source code is currently a mess. Everything is placed in two view controllers. It's almost impossible to fix bugs, let alone add new feature to this codebase. 
3. It seems to me, that only twelve people are really interested in the application. Since I don't use the app personally, I don't think rewriting the app is worth it. 
4. Developing an app without any help is not easy. Contributing to an existing app is easier. The  problem here is, that I need to learn a new programming language before I can contribute to the Flutter app. 
5. I don't know if this application is even compatible anymore with the latest nextcloud cookbook version.

If I ever find the motivation to cleanup the source code, I might reconsider my opinion. The source code is still published under the GPLv3 license. Anybody can contribute, fork the project or upload the app to the app store themselves, as long as they opensource their changes under GPLv3 license. If someone decides to do any of the later it would be great, if he / she informs me, but this is not a requirement according to the license.

**Comment by aniqueta** [comment]
Thanks for sharing your thinking @Schlaubischlump. This makes sense, and unfortunately, I’m not skilled enough to fork or support development. I’ll continue to watch this and related projects though, and my offer to help in other ways stands. All the best.

**Comment by Schlaubischlump** [comment]
Thank you for you kind words and your offer to help. If anything regarding this projects status changes I'll inform you !

**Comment by VincentMeilinger** [comment]
Hey @Schlaubischlump, I'm currently trying to revive your efforts! I think @Teifun2's app is good, but it isn't iOS native and it's missing some features I'd like to have, so I built an iOS Cookbook Client in SwiftUI (it's already on the Appstore!). In case you (or anyone else) are still interested, i'd be happy to work with you!

**Comment by Schlaubischlump** [comment]
Hi @VincentMeilinger,

Sadly I don't have the time at the moment to work on any projects in my free time. Even maintaining LocationSimulator is currently not possible. If this changes anytime soon, I'll let you know!

**Comment by VincentMeilinger** [comment]
Okay, that's understandable, of course! Good luck then with your current obligations!
